.github/workflows/build-and-test.yml

```diff
@@ -99,21 +100,106 @@ jobs:
                   python-version: ${{ matrix.python-version }}
                   channels: conda-forge
 
-            - name: Install dependencies MacOS
+            - name: Install build dependencies MacOS
               if: matrix.os != 'windows-latest'
               shell: bash -l {0}
               run: |
                   conda upgrade -y pip setuptools
-                  conda install Cython requests numpy
+                  pip install toml twine ${{ matrix.numpy }}
+                  pip install $(python -c "import toml;print(' '.join(toml.load('pyproject.toml')['build-system']['requires']))")
+
+            - name: Build wheel
+              shell: bash -l {0}
+              run: |
+                  python setup.py bdist_wheel
+                  python -m twine check dist/*
+
+            - name: Install runtime dependencies Windows
+              if: matrix.os == 'windows-latest'
+              shell: bash -l {0}
+              env:
+                  PIP_EXTRA_INDEX_URL: "http://pypi.naturalcapitalproject.org/simple/"
+                  PIP_TRUSTED_HOST: "pypi.naturalcapitalproject.org"
+                  PIP_PREFER_BINARY: 1
+              run: python -m pip install -r requirements.txt -r requirements-dev.txt
```

Comment:
I just noticed that `requirements-gui` is not included here. The UI tests seem to run okay though, despite the usual flakiness. How exactly does that work? :)

Reply:
Yes indeed!  The GUI tests are executed in a batch separately from the model tests (the `run-ui-tests` job in this configuration file, starting on line 238).  The idea with removing the `-r requirements-gui.txt` was simply to cut down the number of packages that would need to be installed while still being able to run the model tests successfully.  Come to think of it, I could probably remove the `-r requirements-dev.txt` as well, now that build dependencies are tracked in `pyproject.toml` and installed here in the job.

It's also worth noting that the UI tests themselves are still using `python setup.py install` for their installation and not a python artifact.

Reply:
Ah, right, thanks. I missed the ui tests section since nothing was changed there.